src/drafts/SelectPanel2/SelectPanel.tsx

```diff
@@ -243,20 +277,16 @@ const SelectPanelHeader: React.FC<React.PropsWithChildren> = ({children, ...prop
     </Box>
   )
 }
-SelectPanel.Header = SelectPanelHeader
 
-// @ts-ignore todo
-const SelectPanelSearchInput = props => {
+const SelectPanelSearchInput: React.FC<TextInputProps> = ({onChange: propsOnChange, ...props}) => {
```

Comment:
For `onChange`, do they need access to the event itself or could we provide the value of the input instead? (Or a similar `onValueChange`). This might help to simplify the types if `onChange` (or `onValueChange`) is something like `onChange({ value }: { value: string }): void;`